Mobl_ios_Callisto-master/MealTrackingPilot/LogCommonMeal/Views/OccasionPicker.swift

```diff
@@ -531,64 +416,64 @@ open class PickerView: UIView {
         case .vertical:
             offsetPoint.y = offset
         }
-
+        
         if animated {
             isAnimating = true
         }
-
+        
         collectionView.setContentOffset(offsetPoint, animated: animated)
     }
-
+    
     fileprivate var itemSpan: CGFloat {
-        return delegate?.pickerViewSpanForItems(self) ?? 0
+        return delegate?.occasionPickerSpanForItems(self) ?? 0
     }
-
+    
     fileprivate var itemLateralSpan: CGFloat {
         return bounds.size.lateralSpan(forDirection: scrollingDirection)
     }
-
+    
     fileprivate var endCapSpan: CGFloat {
         return (bounds.size.span(forDirection: scrollingDirection) - itemSpan) * 0.5
     }
-
+    
     /**
-        Selects literally the row with index that the user tapped.
-
-        - parameter row: The row index that the user tapped, i.e. the Data Source index times the `infinityItemsMultiplier`.
-    */
+     Selects literally the item with index that the user tapped.
+     
+     - parameter item: The item index that the user tapped, i.e. the Data Source index times the `infinityItemsMultiplier`.
+     */
     fileprivate func selectTappedItem(_ item: Int) {
-        delegate?.pickerView?(self, didTapItem: item, index: indexForItem(item))
+        delegate?.occasionPicker?(self, didTapItem: item, index: indexForItem(item))
         selectItem(item, animated: true)
     }
-
+    
     /**
-        Configure the first row selection: If some pre-selected row was set, we select it, else we select the nearby to middle at all.
-    */
+     Configure the first item selection: If some pre-selected item was set, we select it, else we select the nearby to middle at all.
+     */
     fileprivate func configureFirstSelection() {
         let itemToSelect = currentSelectedItem != nil ? currentSelectedItem : Int(ceil(Float(numberOfItemsByDataSource) / 2.0))
         selectedNearbyToMiddleItem(itemToSelect!)
     }
-
-    fileprivate func turnPickerViewOn() {
+    
+    fileprivate func turnOccasionPickerOn() {
         collectionView.isScrollEnabled = true
     }
-
-    fileprivate func turnPickerViewOff() {
+    
+    fileprivate func turnOccasionPickerOff() {
```

Comment:
Change the filePrivate to private wherever we used

Reply:
Done